### ISSUE TITLE
Change Endpoint to Dynamic

### ISSUE THREAD

**Issue by adamdriscoll** [ironmansoftware/universal-dashboard#1457]
I'm proposing to change the Endpoint terminology to Dynamic. Endpoint makes little sense to new and old users alike. There are two ways of doing this: one breaking and one not. 

One would be to add an alias to `-Endpoint` and call it `-DynamicContent`. This is non-breaking and would start a transition over to the new terminology. 

```
New-UDCard -DynamicContent {
    "Some stuff"
}
```

The second, which is breaking but I like better, is to include a switch parameter on components that support dynamic content. Then you could switch better dynamic and static content by including the switch parameter. I just think it reads better. 

```
New-UDCard -Content {
    "Some stuff"
} -Dynamic
```

Would love to hear comments.

**Comment by adamdriscoll** [comment]
This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:

https://forums.universaldashboard.io/t/changing-endpoint-to-dynamic/2012/1

**Comment by wsl2001** [comment]
I like the second breaking option.

**Comment by adamdriscoll** [comment]
This is now implemented differently than was discussed to greatly reduce the complexity of components. Rather than each component having to have an Endpoint vs Content parameter, I've implemented a New-UDDynamic cmdlet. This cmdlet effectively makes certain sections of a dashboard dynamic. The controls within them don't have to implement dynamic themselves. The entire block will just reload itself. 

New-UDDynamic will support Sync-UDElement, autorefresh and I'm even considering putting some sort of UI for manual refreshes. Any component within a Dynamic section will be able to take advantage of this.
